### ISSUE TITLE
After upgrade to 2024.8.2, unable to use climate.turn_off or turn_on

### ISSUE THREAD

**Issue by OmenWild** [dlarrick/hass-kumo#164]
I have automations with:
```yaml
- service: climate.turn_off
  entity_id: climate.computer_room
```
But after upgrading to 2024.8.2 I get the following error in the log: 
`Entity climate.computer_room does not support this service.`

Additionally, when I try to use `Climate: Turn off` from Developer Tools -> Actions, it shows `No matching entities found` when I try to add an entity. `Climate: Turn on` shows the same lack of entites, but `Climate: Set HVAC mode` shows all my minisplits, matching what I am seeing in the automations.

My current workaround is this:
```yaml
- action: climate.set_hvac_mode
  target:
    entity_id: climate.computer_room
  data:
    hvac_mode: "off"
```

I am currently running the latest released version of this plugin installed with HACS.

**Comment by VeniceNerd** [comment]
I use the "change mode" and "set to off" settings. That seems to work.

**Comment by ilndboy** [comment]
Using 'set hvac mode' to 'off' is not working for me. I have a house full of minisplits that I have to turn off manually. Do we know if there is a fix being worked on for this?

**Comment by dlarrick** [comment]
OK, I finally had some time to research this issue. Here's what I found:

https://developers.home-assistant.io/blog/2024/01/24/climate-climateentityfeatures-expanded/

TL;DR: If an integration supports climate.turn_on / climate.turn_off it needs to set the proper ClimateEntityFeature flags, or a warning is emitted. Pull request https://github.com/dlarrick/hass-kumo/pull/155 fixed this deprecation warning, but did so in a way that disables turn_on / turn_off.

If one of you who sees this issue could test this out by adding `ClimateEntityFeature.TURN_ON` and `ClimateEntityFeature.TURN_OFF` (delimited by `|`) to https://github.com/dlarrick/hass-kumo/blob/master/custom_components/kumo/climate.py#L158 in your local copy and try it out, we can see if that fixes the issue and also still avoids the deprecation warning.

**Comment by ilndboy** [comment]
Thank you for looking into this. I added the code as you instructed and restarted home assistant after doing so. Both my Node Red automation to turn off the heat and the back-up native HA automation I built when I started having this issue are still not turning off the heat when instructed. Both look as though they successfully make the call to turn it off, but when I check the heat is still on.
In Node Red I am trying first trying 'climate.set_hvac_mode' and setting mode to 'off', then delaying a second, then calling 'climate.turn_off' and neither are working.
Let me know if I can help further at all - and thanks again.

**Comment by dlarrick** [comment]
The change in this PR: https://github.com/dlarrick/hass-kumo/pull/163/files is what you tried? And it's not working for you? That user reported success. 

Just for a sanity check, is your main HomeAssistant version reasonably up to date?

If this doesn't work the next step is to look at what the other (officially-supported) climate integrations are doing here. I doubt I'll have time before the weekend.

**Comment by ilndboy** [comment]
Following your instructions I interpreted them in this way:
![image](https://github.com/user-attachments/assets/8518ceb6-7882-423c-980d-dad8473c65e2)
This did not work, but the Kumo integration loaded (I could manually control my heat pumps using the climate cards and see the data).

After reading the post you referenced, I changed it to look like this:
![image](https://github.com/user-attachments/assets/6c14f22d-563f-4831-a119-deb91ee39ed3)
This caused the Kumo integration to not even be able to load correctly (everything was showing as 'unavailable').

**Comment by dlarrick** [comment]
Your earlier one was more correct I think. The 2nd attempt is missing TURN_ON (and has FAN_MODE twice but that should not matter). So let me ask again, what version of HomeAssistant are you running?

**Comment by ilndboy** [comment]
My apologies - I forgot to answer the other questions;
I always update home assistant and all of my integrations/add ons as soon as updates are available - I can't stand seeing the notification icons! I only do stable builds for everything though - no betas.
I edited my file back to the first version and continue monitoring it.
Thanks again for your work here - most of the reason I was willing to go with these heat pumps is because I knew I could use this integration to control them via Home Assistant. I am grateful for your work here.

**Comment by mxr** [comment]
FWIW the integration won't load for the second one because you are missing a `|` at the end of line 158. (But you can remove line 158 altogether too)

**Comment by OmenWild** [comment]
I wanted to confirm that release [v0.3.15](https://github.com/dlarrick/hass-kumo/releases/tag/v0.3.15) resolved this issue for me. Thank you!
